sqlserver/metadata.csv

```diff
@@ -1,11 +1,24 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
 sqlserver.access.page_splits,gauge,,operation,second,The number of page splits per second.,-1,sql_server,page splits,
+sqlserver.ao.is_primary_replica,gauge,,,,"Returns 1 if the replica is primary, or 0 if it is a secondary replica.",0,sql_server,is primary replica,
 sqlserver.ao.ag_sync_health,gauge,,,,"Availability group synchronization health: 0 = Not healthy, 1 = Partially healthy, 2 = Healthy",0,sql_server,ag sync health,
 sqlserver.ao.replica_sync_state,gauge,,,,"Replica synchronization state: 0 = Not synchronizing, 1 = Synchronizing, 2 = Synchronized, 3 = Reverting, 4 = Initializing",0,sql_server,replica sync state,
+sqlserver.ao.log_send_queue_size,gauge,,,,"Amount of log records of the primary database that has not been sent to the secondary databases, in kilobytes (KB).",0,sql_server,log send queue size,
```

Comment:
All KB metrics should be sent as `bytes` (i.e. multiply the column by 1024 or 1000, whichever is appropriate) and the unit should be `byte`. This will allow our platform to handle the conversions at query time.

Take a look at the CSV headers.